sklearn/metrics/pairwise.py

```diff
@@ -958,19 +958,24 @@ def paired_manhattan_distances(X, Y):
 
 def paired_cosine_distances(X, Y):
     """
-    Computes the paired cosine distances between X and Y.
+    Compute the paired cosine distances between X and Y.
 
     Read more in the :ref:`User Guide <metrics>`.
 
     Parameters
     ----------
-    X : array-like of shape (n_samples, n_features)
+    X : array-like of shape (n_samples_X, n_features)
+        An array where each row is a sample and each column is a feature.
 
-    Y : array-like of shape (n_samples, n_features)
+    Y : array-like of shape (n_samples_Y, n_features), default=None
+        An array where each row is a sample and each column is a feature.
+        If `None`, method uses `Y=X`.
```

Comment:
`paired_cosine_distances` does not have a default for `Y`.

Reply:
Ah yeah I was going off of the pairwise signature not the paired. Fixed now. Thanks!